packages/fxa-auth-server/scripts/delete-unverified-accounts/delete-unverified-accounts.ts

```diff
@@ -0,0 +1,308 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Command } from 'commander';
+import { StatsD } from 'hot-shots';
+import PQueue from 'p-queue';
+import { Container } from 'typedi';
+
+import { setupAccountDatabase } from '@fxa/shared/db/mysql/account';
+
+import appConfig from '../../config';
+import * as random from '../../lib/crypto/random';
+import DB from '../../lib/db';
+import { setupFirestore } from '../../lib/firestore-db';
+import initLog from '../../lib/log';
+import { CurrencyHelper } from '../../lib/payments/currencies';
+import { createStripeHelper, StripeHelper } from '../../lib/payments/stripe';
+import initRedis from '../../lib/redis';
+import Token from '../../lib/tokens';
+import { AppConfig, AuthFirestore, AuthLogger } from '../../lib/types';
+import { parseDryRun } from '../lib/args';
+import {
+  AccountTasksFactory,
+  ReasonForDeletion,
+} from '@fxa/shared/cloud-tasks';
+import { getAccountCustomerByUid } from 'fxa-shared/db/models/auth';
+
+const collect = () => (val: string, xs: string[]) => {
+  xs.push(val);
+  return xs;
+};
+const uid = collect();
+const email = collect();
+
+const limitSpecifiedAccounts = (
+  program: Command,
+  limit: number
+): { uids: string[]; emails: string[] } => {
+  if (limit === Infinity) {
+    return { uids: program.uid, emails: program.email };
+  }
+  if (program.uid.size >= limit) {
+    return { uids: program.uid.slice(0, limit), emails: [] };
+  }
+  return {
+    uids: program.uid,
+    emails: program.email.slice(0, limit - program.uid.length),
+  };
+};
+
+const dryRun = (
+  program: Command,
+  useSpecifiedAccounts: boolean,
+  useDateRange: boolean,
+  limit: number
+) => {
+  if (useSpecifiedAccounts) {
+    const { uids, emails } = limitSpecifiedAccounts(program, limit);
+    console.log(
+      `When not in dry-run mode this call will enqueue ${
+        uids.length + emails.length
+      } account deletions with the following account info`
+    );
+    uids.forEach((x) => console.log(`uid: ${x}`));
+    emails.forEach((x) => console.log(`uid: ${x}`));
```

Comment:
Q: should this say `email:` instead of `uid:`?
```suggestion
    emails.forEach((x) => console.log(`email: ${x}`));
```